docs/reference/api.md

```diff
@@ -218,112 +218,6 @@
-
-// OR
-
-fcl.config({
-  foo: "bar",
-  baz: "buz",
-});
-```
-
-### Getting Configuration Values
-
-The `config` instance has an **asynchronous** `get` method. You can also pass it a fallback value.
-
-```javascript
-import * as fcl from "@onflow/fcl";
-
-fcl.config().put("foo", "bar").put("woot", 5).put("rawr", 7);
-
-const FALLBACK = 1;
-
-async function addStuff() {
-  var woot = await fcl.config().get("woot", FALLBACK); // will be 5 -- set in the config before
-  var rawr = await fcl.config().get("rawr", FALLBACK); // will be 7 -- set in the config before
-  var hmmm = await fcl.config().get("hmmm", FALLBACK); // will be 1 -- uses fallback because this isnt in the config
-
-  return woot + rawr + hmmm;
-}
-
-addStuff().then((d) => console.log(d)); // 13 (5 + 7 + 1)
-```
-
-### Common Configuration Keys
-
-| Name                                   | Example                                              | Description                                                                                                                                                                                    |
-| --------------------------------- | --------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `accessNode.api` **(required)**        | `https://access-testnet.onflow.org`                  | API URL for the Flow Blockchain Access Node you want to be communicating with. See all available access node endpoints [here](https://docs.onflow.org/access-api/#flow-access-node-endpoints). |
-| `app.detail.title`                     | `Cryptokitties`                                      | Your applications title, can be requested by wallets and other services.                                                                                                                       |
-| `app.detail.icon`                      | `https://fcl-discovery.onflow.org/images/blocto.png` | Url for your applications icon, can be requested by wallets and other services.                                                                                                                |
-| `challenge.handshake`                  | **DEPRECATED**                                       | Use `discovery.wallet` instead.|
-| `discovery.authn.endpoint`             | `https://fcl-discovery.onflow.org/api/testnet/authn` | Endpoint for alternative configurable Wallet Discovery mechanism. Read more on [discovery](#discovery)                                                                                         |
-| `discovery.wallet` **(required)**      | `https://fcl-discovery.onflow.org/testnet/authn`     | Points FCL at the Wallet or Wallet Discovery mechanism.                                                                                                                                        |
```

Comment:
should add **required** flag for `accessNode.api` and `discovery.wallet ` in configure-fcl.mdx

Reply:
I will resolve it.